heralding/reporting/aux_data_fields.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (C) 2019 Sudipta Pandit <realsdx@protonmail.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+class AuxiliaryData:
```

Comment:
Would it be possible to get rid of this class altogether? 
Knowledge about (csv) field names could be moved to the relevant capability and figuring out the log file name could be moved to the for _p in AuxiliaryData.fields_ loop i file_logger.py.

Reply:
I thought about it before, and i thought aux logging can be enabled for many protocols so it might be better to implement in separate class, as aux logging is an additional thing and having fields of all protocols in file_logger.py will make it unnecessary large.
Anyways, if yo think it should be changed, just let me know.

Reply:
Should i update the PR ?